### PR TITLE
VGG16 Training Notebook with 0.947 Validation Accuracy

### PR DESCRIPTION
**Pull request by hemhemoh**
Hi, With respect to issue #178 that was assigned to me, I am done with the training of the model. I used the VGG16 pre-trained model and I had 94.7% accuracy. However, the tflite version of the model is more than 100MB, so I can upload it here. Do I go ahead to upload the pkl version of the model only?

### CODE REVIEW

**Comment by freny24** [comment]
@hemhemoh hey ! can you upload a python script instead of ipynb notebook? It will be easier to integrate it into flask

**Comment by hemhemoh** [comment]
> @hemhemoh hey ! can you upload a python script instead of ipynb notebook? It will be easier to integrate it into flask

Done that now

**Review by hemhemoh — COMMENTED** [review]
Hi, I have committed the .py notebook and also pushed my model to the model folder. Kindly review

**Comment by freny24** [comment]
@hemhemoh Excellent work!!

**Comment by freny24** [comment]
@hemhemoh you PR is merged! Sorry for the delay:)

**Comment by freny24** [comment]
@hemhemoh  You were ready with your new PR right? Do create it and send it in discord so that I can check

**Comment by hemhemoh** [comment]
@freny24 I didn't know I have to create 2 separate pull requests. Committed changes for the second one before you merged the pull request, so you merged the 3 already. I was thinking if you could help me upgrade the task to level 3, so I wont miss out too much points

**Comment by freny24** [comment]
@hemhemoh oh that's fine if you merged it into one ! Also , I have already given the label of Level3 for this PR

**Comment by hemhemoh** [comment]
> @hemhemoh oh that's fine if you merged it into one ! Also , I have already given the label of Level3 for this PR

@freny24 okay, thank you
